core/src/main/resources/plugin.yml

```diff
@@ -1,8 +1,9 @@
 name: LWC
 main: com.griefcraft.lwc.LWCPlugin
-version: ${project.version}
+version: 4.5.1-SNAPSHOT
```

Comment:
Yep, and they both display the website as well, even with your latest changes pulled in that remove it from plugin.yml.